src/pages/test-report/test-report.html

```diff
@@ -15,7 +15,7 @@
     padding-left
     padding-right
     class="grid-layout"
-    [ngClass]="isSeriousMode ? 'serious-mode' : isDangerousMode ? 'dangerous-mode' : ''">
+    [ngClass]="isRemoveFaultMode ? 'remove-mode' : isSeriousMode ? 'serious-mode' : isDangerousMode ? 'dangerous-mode' : ''">
```

Comment:
This now looks very untidy. Perhaps write a method that returns the class based on the mode and call that.